app/package.json

```diff
@@ -235,7 +236,7 @@
     "storybook-addon-jsx": "^5.4.0",
     "storybook-chromatic": "^1.2.0",
     "storybook-react-router": "^1.0.1",
-    "streamr-client": "^2.0.0-beta.7",
+    "streamr-client": "^2.2.2",
```

Comment:
FYI @timoxley the new client will require some changes in the editor, this can be reproduced when a canvas is started:

<img width="1650" alt="Screenshot 2019-10-15 at 10 29 23" src="https://user-images.githubusercontent.com/1593398/66809806-c1c9a180-ef36-11e9-8a7c-d9e28c32ae14.png">

Reply:
Oh crap, I didn't test editor at all after this change 😞I had problems with 2.0.0 client but maybe we can work around them if editor changes are too big.